### ISSUE TITLE
Delay in doing moves

### ISSUE THREAD

**Issue by skysplatwoomy** [danieljabailey/giant_led_cube#23]
If I were to do R, then R' and repeat in somewhat quick succession, the input is delayed which is bad for things like sexy move.

**Comment by danieljabailey** [comment]
Yeah, this is something I wanted to tweak before I arrived and set up the cube but ran out of time, unfortunately I can't reprogram the Pico without opening up the cube and plugging my laptop in :'(
It's the sightly over-enthusiastic debounce code that is required because the paddles can bounce back and hit the opposite switch when you let them spring back.
I think it's a 500ms delay currently, and you're right, that's too long. Unfortunately I'll probably not find the enthusiasm to fix it this weekend, considering how awkward it is to open the cube up.
